### PR TITLE
Digiflag v2.5.1

### PR DESCRIPTION
**Pull request by TheBaconCat**
Pull Request to Update Digiflag to v2.5.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<img src="./src/renderer/assets/logos/DigiFlag.png" alt="DigiFlagLogo" width="128"/><img src="https://user-images.githubusercontent.com/11319293/218509478-f1cf9819-560e-4aa4-bdba-09eec2c8f264.png" alt="Vite" width="128"/>
+<img src="https://github.com/LapsTimeOFF/DigiFlag_F1MV/raw/stable/src/renderer/assets/logos/DigiFlag.png" alt="Vite" width="128"/> <img src="https://user-images.githubusercontent.com/11319293/218509478-f1cf9819-560e-4aa4-bdba-09eec2c8f264.png" alt="Vite" width="128"/>
```
Comment (LapsTimeOFF) [inline README.md]:
```suggestion
<img src="https://github.com/LapsTimeOFF/DigiFlag_F1MV/raw/stable/src/renderer/assets/logos/DigiFlag.png" alt="DigiFlag" width="128"/> <img src="https://user-images.githubusercontent.com/11319293/218509478-f1cf9819-560e-4aa4-bdba-09eec2c8f264.png" alt="Vite" width="128"/>
```

File: src/renderer/public/localization/locales/es/translation.json [hunk 2]
```diff
@@ -1,15 +1,18 @@
 {
+  "alwaysOnTop": "Always on Top?",
```
Comment (LapsTimeOFF) [inline src/renderer/public/localization/locales/es/translation.json]: To be translated?

File: src/renderer/public/localization/locales/fr/translation.json [hunk 3]
```diff
@@ -1,15 +1,18 @@
 {
+  "alwaysOnTop": "Always on Top?",
```
Comment (LapsTimeOFF) [inline src/renderer/public/localization/locales/fr/translation.json]:
```suggestion
  "alwaysOnTop": "Toujours au dessus?",
```

File: src/renderer/public/localization/locales/fr/translation.json [hunk 4]
```diff
@@ -1,15 +1,18 @@
...
   "attemptingToConnectToF1MV": "Tentative de connexion à F1MV dans:",
   "attemptingToConnectToLiveReplayTimingWindow": "Tentative de connexion à la fenêtre de télémétrie en direct ou de rediffusion:",
```
Comment (LapsTimeOFF) [inline src/renderer/public/localization/locales/fr/translation.json]:
```suggestion
  "attemptingToConnectToLiveReplayTimingWindow": "Tentative de connexion au Live Timing :",
```

File: src/renderer/public/localization/locales/fr/translation.json [hunk 5]
```diff
@@ -1,15 +1,18 @@
...
   "attemptingToGetCurrentSessionInfoFromF1mv": "Récupération des informations de la session actuelle de F1MV...",
   "back": "Retourner",
   "checkNetworkSettings": "Déplacez votre souris dans le coin inférieur droit et cliquez sur la roue des paramètres pour vérifier vos paramètres réseau.",
```
Comment (LapsTimeOFF) [inline src/renderer/public/localization/locales/fr/translation.json]:
```suggestion
  "checkNetworkSettings": "Déplacez votre souris dans le coin inférieur droit et cliquez sur le bouton paramètres pour vérifier vos paramètres réseau.",
```

File: src/renderer/public/localization/locales/es/translation.json [hunk 6]
```diff
@@ -1,15 +1,18 @@
...
   "back": "Atrás",
   "checkNetworkSettings": "Mueva el ratón a la esquina inferior derecha y haga clic en el engranaje de configuración para comprobar la configuración de red.",
   "currentSession": "Sesión en Curso:",
   "digiflagNotLinked": "DigiFlag no está vinculado actualmente a F1MV",
+  "dutch": "Holandés",
   "english": "Inglés",
   "failedToConnect": "Fallo de conexión con F1MV",
   "failedToRetrieveCurrentSession": "Fallo al recuperar la sesión actual",
   "french": "Francés",
+  "general": "General",
```
Comment (LapsTimeOFF) [inline src/renderer/public/localization/locales/es/translation.json:15]: To be translated?

File: src/renderer/public/localization/locales/fr/translation.json [hunk 7]
```diff
@@ -1,15 +1,18 @@
...
   "currentSession": "Session Actuelle:",
   "digiflagNotLinked": "DigiFlag n'est actuellement pas lié à F1MV",
+  "dutch": "Néerlandais",
   "english": "Anglais",
   "failedToConnect": "Impossible de se connecter à F1MV",
   "failedToRetrieveCurrentSession": "Impossible de récupérer la session actuelle",
   "french": "Français",
+  "general": "Général",
   "infoTag": "Peut-être essayez-vous de vous connecter à une autre hôte ? Peut-être que votre port n'est pas celui par défaut ?",
```
Comment (LapsTimeOFF) [inline src/renderer/public/localization/locales/fr/translation.json]:
```suggestion
  "infoTag": "Peut-être essayez-vous de vous connecter à un autre ordinateur ? Peut-être que votre port n'est pas celui par défaut ?",
```

File: src/renderer/public/localization/locales/nl/translation.json [hunk 8]
```diff
@@ -0,0 +1,44 @@
+{
+  "alwaysOnTop": "Always on Top?",
```
Comment (LapsTimeOFF) [inline src/renderer/public/localization/locales/nl/translation.json]: To be translated?

**Review by LapsTimeOFF — CHANGES_REQUESTED** [review]
Some small changes to be done, I've tried to use the "Suggestion" thing of Github so you can commit directly

File: src/renderer/public/localization/locales/es/translation.json (hunk 2 above)
Comment (TheBaconCat) [reply src/renderer/public/localization/locales/es/translation.json]: Yep I couldn’t find the right wording that made sense

File: src/renderer/public/localization/locales/es/translation.json (hunk 6 above)
Comment (TheBaconCat) [reply src/renderer/public/localization/locales/es/translation.json:15]: No that’s actually correct I was surprised too

File: src/renderer/public/localization/locales/nl/translation.json (hunk 8 above)
Comment (TheBaconCat) [reply src/renderer/public/localization/locales/nl/translation.json]: Yes to be translated

File: src/renderer/public/localization/locales/es/translation.json (hunk 2 above)
Comment (LapsTimeOFF) [reply src/renderer/public/localization/locales/es/translation.json]:
```suggestion
  "alwaysOnTop": "Siempre en la cima?",
```

File: src/renderer/public/localization/locales/nl/translation.json (hunk 8 above)
Comment (LapsTimeOFF) [reply src/renderer/public/localization/locales/nl/translation.json]:
```suggestion
  "alwaysOnTop": "Altijd bovenop?",
```

**Review by LapsTimeOFF — APPROVED** [review]
LGTM

**Pull request merged**